src/query/config/src/config.rs

```diff
@@ -1805,7 +1818,8 @@ impl From<InnerQueryConfig> for QueryConfig {
             jwt_key_files: inner.jwt_key_files,
             default_storage_format: inner.default_storage_format,
             default_compression: inner.default_compression,
-            users: users_from_inner(inner.idm.users),
+            users: inner.idm.users,
+            udfs: inner.idm.udfs,
```

Comment:
Since `idm` means `identity manager`, maybe `udfs` should be locationed outside of it?

Reply:
I plan to change the name from `idm` to `builtin`.

Reply:
what about `predefined`  or `preconfigured`?  `built-in` sounds like `hard-coded` in code?

Reply:
@BohuTANG